prow/config/jobs_test.go

```diff
@@ -180,7 +180,8 @@ func TestCommentBodyMatches(t *testing.T) {
 		Presubmits: map[string][]Presubmit{
 			"org/repo": {
 				{
-					Name:      "gce",
+					Name: "gce",
+					//should we change this to: /test <some-test-name> ?
```

Comment:
@fejta I remembered that you suggested to not use new bot command format for testing in this file. As the old format will be retired, i think we can now use the new format here.